src/hb-ot-var-fvar-table.hh

```diff
@@ -173,17 +164,8 @@ struct fvar
   {
     if (axes_count)
     {
-      /* TODO Rewrite as hb_array_t<>::sub-array() */
-      unsigned int count = axisCount;
-      start_offset = hb_min (start_offset, count);
-
-      count -= start_offset;
-      axes_array += start_offset;
```

Comment:
This is the part that was wrong, start_offset isn't intended to be added to axes_array index but just used just as a AxisRecord index offset.